projects/js-packages/connection/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/jetpack-connection",
-	"version": "0.13.0-alpha",
+	"version": "0.13.1-alpha",
```

Comment:
Something seems to have gone wrong here, a release PR should be releasing 0.13.1 rather than going directly to 0.13.1-alpha.